Description: Account for missing default feature of rustls in Debian
 Needs another debcargo invocation to take effort.
Forwarded: not-needed
Last-Update: 2025-03-28
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -173,2 +173,4 @@
 version = "0.23"
+default-features = false
+features = ["ring", "logging", "std", "tls12"] # default aws_lc_rs unavailable in Debian
 optional = true
